### PR TITLE
Added logo files

### PR DESCRIPTION
**Pull request by ivanha09**
Added logo files and added logo to the readme.md file. It should show up once the the files are uploaded.

### CODE REVIEW

**Comment by ybiquitous** [comment]
@ivanha09 Could you please add `.DS_Store` line to the `.gitignore` file and remove `.DS_Store` files from this PR?

https://github.com/sider/goodcheck/blob/986e6899d67469789789c9f2e89e4501010faddc/.gitignore#L11

For example:

```diff
 /Gemfile.lock
+.DS_Store
```

`.DS_Store` files are not committed usually. 😅

**Comment by ivanha09** [comment]
Ok. Thank you!

**Comment by ivanha09** [comment]
@ybiquitous After try it out. I think I finally got it correct. Will it merge properly now?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+![Goodcheck logo](https://raw.githubusercontent.com/sider/goodcheck/master/logo/GoodCheck%20Horizontal.png)
```
Comment (ybiquitous) [inline README.md]: Relative path is available here! 😃 
If doing so, the image will be shown corretly on viewing README. 👍 

```suggestion
![Goodcheck logo](logo/GoodCheck%20Horizontal.png)
```

**Comment by ybiquitous** [comment]
> After try it out. I think I finally got it correct.

Thanks! I confirmed `.gitignore` and removed `.DS_Store` files. 👍 

> Will it merge properly now?

Actually, I don't know this PR's background, so I can't whether should merge it or not... 🤷‍♂️

@soutaro Will you review it?

File: README.md (hunk 1 above)
Comment (ivanha09) [reply README.md]: Thanks! That makes more sense! 😃

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+![Goodcheck logo](logo/GoodCheck%20Horizontal.png)
```
Comment (soutaro) [inline README.md]: 👍

**Pull request merged**